### ISSUE TITLE
Can't override some fields attributes and form query parameters in Freeform5

### ISSUE THREAD

**Issue by valboivin** [solspace/craft3-freeform#1028]
**Problematic** 

After upgrading to Freeform 5, I can't seem to be able to override some attributes or parameters.

- The `fieldIdPrefix` parameter doesn't seem to be supported when we use it in the form query. It doesn't prefix any of my fields `id`.
- I can't override the `class` of the error list. I've tried different ways to change the `class` of either the inputs with an error or the error list itself and none seem to be working. (see example below)

**Steps to reproduce**

```
{{ freeform.form(form.form.handle, {
    fieldIdPrefix: form.id ~ '-'
    }).render({
        attributes: {
            id: 'form-' ~ form.id
        },
        fields: {
            "@global": {
                attributes: {
                    error: {
                        "=class": "errors"
                    }
                },
            },
            ":errors": {
                attributes: {
                    input: { "=class": "is-invalid has-validation" },
                },
            },
        }
    }) 
}}
```
**Expected behavior**

- All fields `id` should be prefixed with my custom form id (i.e. `form-19856-`)
- The `ul` representing the errors list should have a class of `errors` and not the one defined by the Bootstrap 5 templates. (The one used by our form in the CMS)

**Craft & Plugin Info**

- Craft Version: 4.7.1
- Freeform Version: 5.0.7
- Freeform Edition: Pro
- Upgraded from: 4.1.13

**Comment by kjmartens** [comment]
Sorry for the trouble @valboivin.

We'll have a developer check into the `fieldIdPrefix` issue... it was probably overlooked. 🙂 

As for the error class... is this to be applied to the field? Or the form error banner? If using AJAX, you'll need to apply this with the JS override approach instead. Something to the effect of:

``` js
<script>
var form = document.querySelector('[data-id="{{ form.anchor }}"]');
if (form) {
    form.addEventListener('freeform-ready', function(event) {
        var freeform = event.target.freeform;
        freeform.setOption("errorClassBanner", ["alert", "alert-danger"]);
        freeform.setOption("errorClassList", ["list-unstyled", "m-0", "fst-italic", "text-danger"]);
        freeform.setOption("errorClassField", ["is-invalid"]);
        freeform.setOption("successClassBanner", ["alert", "alert-success"]);
    });
}
</script>
```

**Comment by valboivin** [comment]
Oh thank you @kjmartens, since some of my other overwrites worked, I didn't understand that others could only be changed with JS for AJAX forms.

**Comment by kjmartens** [comment]
Yes it can be a bit confusing, but the ones based on AJAX interaction (errors, success) must be handled with JS. 🙂

**Comment by valboivin** [comment]
I see! Thanks for the clarification. It does make sense now! 😀

**Comment by kjmartens** [comment]
The `fieldIdPrefix` parameter issue is fixed in Freeform 5.0.9 now. 🙂
